src/main/java/org/truffleruby/parser/YARPMultiTargetNodeTranslator.java

```diff
@@ -113,6 +107,18 @@ public MultipleAssignmentNode translate() {
         return multipleAssignmentNode;
     }
 
+    private AssignableNode[] processAssignables(Nodes.Node[] nodes) {
+        if (nodes.length == 0) {
+            return MultipleAssignmentNode.EMPTY_ASSIGNABLES;
```

Comment:
Should be `AssignableNode.EMPTY_ARRAY`, that's why CI fails, but @andrykonchin already fixed it.

Reply:
Thanks. I had the change locally but forgot to add it to the commit, so it passed locally, but obviously would fail CI.